enforcerlbp/enforcerlbp.sln

```diff
@@ -1,9 +1,21 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.32228.343
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32630.192
```

Comment:
Are you aware of this `VisualStudioVersion = 17.2.32630.192` change?

Reply:
No Sir, how do I resolve this?

Reply:
Do a research on the change and find out if there is any implications then we will if we should revert or just allow it

Reply:
ok sir